.github/workflows/ci.yml

```
@@ -97,6 +97,24 @@ jobs:
       - name: ⬆️ Upload coverage report
         uses: codecov/codecov-action@v4
 
+  are-the-types-wrong:
+    name: 🤔 Are the types wrong?
+    runs-on: ubuntu-latest
+    steps:
+      - name: ⬇️ Checkout repo
+        uses: actions/checkout@v4
+
+      - name: ⎔ Setup Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20
```

Let's keep this at 18, the same as the release part

```suggestion
          node-version: 18
```